--- vstudio.py.orig     2022-07-21 16:11:41.344464416 +0000
+++ vstudio.py  2022-07-21 16:12:50.169360429 +0000
@@ -557,7 +557,7 @@
         self.descriptionDone = 'built ' + self.describe_project()
         yield self.updateSummary()

-        command = ('FOR /F "tokens=*" %%I in (\'vswhere.exe -property  installationPath\') '
+        command = ('FOR /F "tokens=*" %%I in (\'vswhere.exe -products * -property  installationPath\') '
                    f' do "%%I\\%VCENV_BAT%" x86 && msbuild "{self.projectfile}" '
                    f'/p:Configuration="{self.config}" /p:Platform="{self.platform}" /maxcpucount')
